shippers/composite.py

```diff
@@ -64,7 +65,10 @@ def send(self, event: dict[str, Any]) -> str:
             return EVENT_IS_FILTERED
 
         for shipper in self._shippers:
-            shipper.send(event)
+            # dict are mutated if not deep copied, every shipper can mutate the
```

Comment:
the changes in this file are related to the second bugfix:
- Fix events mutation across different outputs of the same input

`shipper.send()` is expected to mutate the `event` we pass to it: in the case the `CompositeShipper` composes multiple shippers, the mutation will spread across the shippers (like carrying metadata or similar specific to a shipper: see an example of the bug where the tag related to the dataset added by the es shipper is still hold on the logstash shipper event).